app/components/file_attachment_list/file_attachment_list_container.js

```diff
@@ -31,4 +31,4 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(makeMapStateToProps, mapDispatchToProps)(FileAttachmentList);
+export default connect(makeMapStateToProps(), mapDispatchToProps)(FileAttachmentList);
```

Comment:
Yeah you're right, I was changing that up to try some other things and didn't put it back 😄  I'll fix it.